### ISSUE TITLE
[Question] Is HalpoPlayer dead?

### ISSUE THREAD

**Issue by Optekah** [halpz/HalpoPlayer#21]
Hey, just checking back in since my last visit back in 2023. I have a handful of enhancements, suggestions, and bugs that come to mind, but from what I can tell, all development has been ceased for many months now. Not sure how much drive I have to open a bunch of issues if this the end of the line.

Is there any plans at all to continue on with this project?

Thanks!

**Comment by ema07cqc** [comment]
Yeah it seems like the creator is focused on other things because the project looks abandoned. It’s a shame because the app is great, I use it all the time; but of course it needs some refinements.

**Comment by mbaiti** [comment]
So sad. Would love to see further development!

**Comment by snppls** [comment]
I just discovered this app after testing different navidrome players for the last few months. This is the only one that nails the user interface for a music app. Editing playlists and adding songs to the queue is all that is needed to make it perfect. Would love for development to continue!!!

**Comment by halpz** [comment]
Hello! it's not dead - I've just had a busy few months - but I do intent to return to it !
